demo/server/config-dev.json

```diff
@@ -19,5 +19,9 @@
   },
   "redis": {
     "url": "redis://127.0.0.1:6379"
+  },
+  "jwtSecret": "raincatcher",
```

Comment:
I will suggest here to put really long generated hash (min 250bits).
It's also important to document this properly.